src/System.Drawing.Common/src/System/Drawing/Drawing2D/GraphicsPath.Unix.cs

```diff
@@ -203,7 +203,7 @@ public byte[] PathTypes
                 int status = Gdip.GdipGetPointCount(_nativePath, out count);
                 Gdip.CheckStatus(status);
                 if (count == 0)
-                    throw new ArgumentException("PathTypes");
+                    throw new ArgumentException(nameof(PathTypes));
```

Comment:
Not for this PR, but we should revisit these two ArgumentExceptions separately... the first argument to the ctor is unfortunately not the name of the parameter.